cluster/gce/config-test.sh

```diff
@@ -399,6 +399,7 @@ if [[ -n "${LOGROTATE_MAX_SIZE:-}" ]]; then
 fi
 
 # Fluentd requirements
+FLUENTD_GCP_VERSION="${FLUENTD_GCP_VERSION:-0.2-1.5.28-1}"
```

Comment:
Same here - this needs adding to PROVIDER_VARS. I guess we should refactor this instead of having two copies...

Reply:
Oops, updated!